_includes/tutorials/versioned-ktables/kstreams/markup/dev/make-build-file.adoc

```diff
@@ -0,0 +1,5 @@
+Create the following Gradle build file, named `build.gradle` for the project:
+
++++++
+<pre class="snippet"><code class="groovy">{% include_raw tutorials/versioned-ktables/kstreams/code/build.gradle %}</code></pre>
++++++
```

Comment:
sorry I missed this on my first review - Suggest either pointing out here that the `sourceCompatibility` and `targetCompatibility` are set to Java 17 so ensure `java -version` shows version 17 before proceeding, OR do nothing here and instead add Java 17 to the Prerequisites

Reply:
Good suggestion - I've added something to that effect